schemas/aquifer_system.py

```diff
@@ -0,0 +1,32 @@
+from typing import List
+
+from pydantic import BaseModel
+
+from schemas import BaseResponseModel
+
+
+# ------ RESPONSE ----------
+class GeoJSONGeometry(BaseModel):
+    """
+    Geometry schema for GeoJSON response.
+    """
+
+    type: str
+    coordinates: (
+        List[float]
+        | List[List[float]]
+        | List[List[List[float]]]
+        | List[List[List[List[float]]]]
+    )
+
+
+class AquiferSystemResponse(BaseResponseModel):
+    """
+    Response schema for aquifer system details.
+    """
+
+    name: str
+    description: str | None = None
+    aquifer_type: str
+    geographic_scale: str
+    boundary: GeoJSONGeometry | None = None
```

Comment:
the boundary is defined as a multypolygon in the database model. should this just be a string?